src/Map.js

```diff
@@ -88,11 +91,41 @@ function addTripPolys(map) {
   return vehicleBounds;
 }
 
+/*
+ * Creates the map object using a journeySharing location
+ * provider.
+ */
+function initializeMapObject(element) {
+  // In a more normal implementation authTokenFetcher
+  // would actually be making a RPC to a backend to generate
+  // the jwt.  For debugging use cases the jwt gets bundled into
+  // the extracted log data.
+  function authTokenFetcher(options) {
+    console.log("Ignoring options using prebuilt jwt", options);
+    const authToken = {
+      token: jwt,
+      expiresInSeconds: 3300, // TODO #25 - bake in actual expiration time
```

Comment:
Google servers tend to reject any tokens with an expiration other than 3600 seconds.

Reply:
I'd just get rid this field entirely. IIRC it's optional when using the IAM signer endpoint.

Reply:
Done